sound/soc/intel/boards/sof_sdw.c

```diff
@@ -1133,17 +1135,19 @@ static int sof_card_dai_links_create(struct snd_soc_card *card)
 		hdmi_num = SOF_PRE_TGL_HDMI_COUNT;
 
 	/* enable dmic01 & dmic16k */
-	if (sof_sdw_quirk & SOC_SDW_PCH_DMIC || mach_params->dmic_num) {
-		if (ctx->ignore_internal_dmic)
-			dev_warn(dev, "Ignoring PCH DMIC\n");
-		else
-			dmic_num = 2;
+	if (ctx->ignore_internal_dmic) {
+		dev_warn(dev, "Ignoring internal DMIC\n");
+		mach_params->dmic_num = 0;
```

Comment:
Can we be more verbose here in the logs. i.e. what is the impact to the impact to the user and will mic capture still work.
My feeling is that if the average user read this in dmesg they could log an issue or worry that somehow the driver/hw has an issue.

Reply:
I certainly have no objection to tweaking the message, if you would prefer a slightly less alarming message.

Reply:
Changed to `dev_dbg(dev, "SoundWire DMIC is used, ignoring internal DMIC\n");`